packages/iTwinUI-react/package.json

```diff
@@ -47,11 +47,12 @@
     "@itwin/itwinui-css": "^0.63.0",
     "@itwin/itwinui-icons-react": "^1.10.1",
     "@itwin/itwinui-illustrations-react": "^1.3.1",
-    "@tippyjs/react": "^4.2.5",
+    "@tippyjs/react": "^4.2.6",
```

Comment:
Just curious, is there a reason (e.g. fixes something) for making this upgrade to tippyjs/react? Or would just upgrading tippy.js (not react) from "6.3.1 -> 6.3.7" fix the problem you were mentioning?

Reply:
@rohan-kadkol there is some kind of fix for dynamic singleton headless content: https://github.com/atomiks/tippyjs-react/commit/d4c915de38b246d87abfad576887063d0a09d463 . It is unrelated, but I see no reason keeping the outdated version. It's a related package, so why not upgrade?

Regarding tippy.js package, I think the issue was fixed in version 6.3.3 with this commit: https://github.com/atomiks/tippyjs/commit/22481d7655b6eed7884ce84daef52471d526d1dc